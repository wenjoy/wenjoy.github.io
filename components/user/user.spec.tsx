import { render } from '@testing-library/react'
import { User, Status, Role } from '.';

describe('User', async() => {
  it('should renders User correctly', () => {
    const { container } = render(<User name="David" status={Status.Pending} role={Role.Moderator} />)
 expect(container).toMatchSnapshot() })
 })