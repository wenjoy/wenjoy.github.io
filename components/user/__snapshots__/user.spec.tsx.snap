// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User should renders User correctly 1`] = `
<div>
  <div>
    ðŸ¤”
     
    David
     (
    moderator
    ) 
  </div>
</div>
`;
